endpoints.tf

```diff
@@ -0,0 +1,15 @@
+data "aws_vpc_endpoint_service" "transferserver" {
```

Comment:
maybe nitpicking, but can instances of `transferserver` be changed to `transfer_server` please?

Reply:
I thought it was specified as one word in the CLI but its indeed seperated by a "." indicating 2 words. Changed this to transfer_server.